src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/plugin/set_preprocess.cpp

```diff
@@ -2,75 +2,91 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
```

Comment:
`[set_preprocess.cpp](https://github.com/openvinotoolkit/openvino/pull/21131/files#diff-0c956ed4ac2c11d5aac210081c548e79d237e7e211994ba68a4bcbea1379c3b6)` is supposed to work on API 1.0
Let's drop such tests at all

Reply:
Test is removed

Reply:
I reverted deletion to unblock merge because openvino_contrib and npu tests depend on it